buildconfig/CMake/CppCheckSetup.cmake

```diff
@@ -30,7 +30,6 @@ if ( CPPCHECK_EXECUTABLE )
         Framework/Kernel/inc
         Vates/VatesAPI/inc
         Vates/VatesSimpleGui/ViewWidgets/inc
-        Vates/VatesSimpleGui/StandAloneExec/inc
```

Comment:
We noticed that too and @mantid-roman fixed them in #20291, which is also ready to go.